utils/iHAMOCC/prepare_Ndeposition/prepare_N-deposition_input.sh

```diff
@@ -0,0 +1,292 @@
+      WETNHx=wetnhx_input4MIPs_surfaceFluxes_ScenarioMIP_NCAR-CCMI-ssp434-1-0_gn_201501-210012.nc
+      WETNOy=wetnoy_input4MIPs_surfaceFluxes_ScenarioMIP_NCAR-CCMI-ssp434-1-0_gn_201501-210012.nc
+      STARTYEAR=2015
+      ENDYEAR=2100
+    elif [ ${TP} == 'SSP460' ]; then
+      YNAME="201501-210012-ssp460"
+      DRYNHx=drynhx_input4MIPs_surfaceFluxes_ScenarioMIP_NCAR-CCMI-ssp460-1-0_gn_201501-210012.nc
+      DRYNOy=drynoy_input4MIPs_surfaceFluxes_ScenarioMIP_NCAR-CCMI-ssp460-1-0_gn_201501-210012.nc
+      WETNHx=wetnhx_input4MIPs_surfaceFluxes_ScenarioMIP_NCAR-CCMI-ssp460-1-0_gn_201501-210012.nc
+      WETNOy=wetnoy_input4MIPs_surfaceFluxes_ScenarioMIP_NCAR-CCMI-ssp460-1-0_gn_201501-210012.nc
+      STARTYEAR=2015
+      ENDYEAR=2100
+    elif [ ${TP} == 'SSP534os' ]; then
+      YNAME="201501-210012-ssp534os"
+      DRYNHx=drynhx_input4MIPs_surfaceFluxes_ScenarioMIP_NCAR-CCMI-ssp534os-1-0_gn_201501-210012.nc
+      DRYNOy=drynoy_input4MIPs_surfaceFluxes_ScenarioMIP_NCAR-CCMI-ssp534os-1-0_gn_201501-210012.nc
+      WETNHx=wetnhx_input4MIPs_surfaceFluxes_ScenarioMIP_NCAR-CCMI-ssp534os-1-0_gn_201501-210012.nc
+      WETNOy=wetnoy_input4MIPs_surfaceFluxes_ScenarioMIP_NCAR-CCMI-ssp534os-1-0_gn_201501-210012.nc
+      STARTYEAR=2015
+      ENDYEAR=2100
+    elif [ ${TP} == 'SSP585' ]; then
+      YNAME="201501-210012-ssp585"
+      DRYNHx=drynhx_input4MIPs_surfaceFluxes_ScenarioMIP_NCAR-CCMI-ssp585-1-0_gn_201501-209912.nc
+      DRYNOy=drynoy_input4MIPs_surfaceFluxes_ScenarioMIP_NCAR-CCMI-ssp585-1-0_gn_201501-209912.nc
+      WETNHx=wetnhx_input4MIPs_surfaceFluxes_ScenarioMIP_NCAR-CCMI-ssp585-1-0_gn_201501-209912.nc
+      WETNOy=wetnoy_input4MIPs_surfaceFluxes_ScenarioMIP_NCAR-CCMI-ssp585-2-0_gn_201501-209912.nc
+      STARTYEAR=2015
+      ENDYEAR=2100
+    fi
+    OUTFILE=${basename}_${YNAME}_${GRID}_${version}.nc
+    CHECKTRY=ndep_${YNAME}_${GRID}_
+    CHECKFILE=`ls ./${CHECKTRY}*.nc`
+    echo "  ${GRIDFILE}"
+    echo "  ${DRYNHx}"
+    echo "  ${STARTYEAR}"  
+    echo "  ${ENDYEAR}"
+    echo "  ${OUTFILE}"  
+    echo "  CHECK: ${CHECKFILE}"  
+
+
+    # pi-control input4MIPs files downloaded from esgf: https://esgf-node.llnl.gov/search/input4mips/
+    
+    
+    # dry nhx & noy: kg/m2/s
+    # wet nhx & noy: kg/m2/s
+    
+    # Constant for unit conversion:
+    # iHAMOCC units for the input file: kmol N m-2 yr-1
+    # Possible options for unit conversions:
+    # - considering no leap years, but mol-weight 14.01
+    #   s -> y: 365*86400 s/yr
+    #   kg -> kmol: 1/14.01
+    #   c = 2250963.597
+    # - considering leap years (probably would be the most correct one)
+    #   s -> yr: 365.25 * 86400
+    #   kg -> kmol: 1/14.01
+    #   c: 2252505.3533190577
+    # - considering leap years, but take mol-weight of 14:
+    #   s -> yr: 365.25 * 86400
+    #   kg -> kmol: 1/14.
+    #   c: 2254114.285714286
+    # - not considering leap years and take mol-weight of 14:
```

Comment:
Would be worth noting that not considering leap years is correct, since the input4MIPs files are on a 365-day calendar. And maybe just remove the other options, since this is confusing?